apps/e2e/test/e2e_test.exs

```diff
@@ -215,6 +215,16 @@ defmodule E2ETest do
         Enum.each(expected, fn app -> assert [ds.id, app] in actual end)
       end)
     end
+
+    test "events have been stored in estuary" do
+      table = Application.get_env(:estuary, :table_name)
+
+      eventually(fn ->
+        [[actual]] = query("SELECT count(1) FROM #{table}")
+
+        assert actual > 0
```

Comment:
Anyway to have this be a specific number instead of > 0?

Reply:
We don't know how many events the system will create (and this will probably increase over time). In my opinion all we care about is at least one event has been written to the table.
I'm open to alternatives.

Reply:
Yeah, we're got foot gun blasted by a specific count check in e2e just today, so I'd advise we be a little foot gun shy about that

Reply:
I figured as much.  Fine with me for now